### ISSUE TITLE
anti DFS-70 antibody

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11638]
**Preferred term label:**
anti DFS-70 antibody

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001539
- label: Anti-DFS70 antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:23304189
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against DFS70.
- comment: DFS70 IS a nuclear protein called DFS70 (also known as LEDGF/p75, lens epithelium-derived growth factor), which is named for the dense fine speckled (DFS) pattern it produces on HEp-2 indirect immunofluorescence. Anti-DFS70 antibodies have been found in the sera of patients with a variety of conditions including cancer, and even more interestingly in healthy individuals. Anti-DFS70 antibodies are common among ANA-positive individuals with no evidence of systemic autoimmune rheumatic diseases. Among autoimmune patients with this autoantibody over a half had evidence of autoimmune thyroiditis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11638
- synonyms:
